src/components/PageReady/index.js

```diff
@@ -1,15 +1,12 @@
 import React from 'react';
-import axios from 'axios';
 import { interpretErrorMessage } from './formatErrors';
 import type RequestShape from './request';
 
 import defaultRequest from './request';
 
 type PropShape = {
   predefinedAction?: Object,
-  url?: string,
-  postBody?: Object,
-  method?: string,
+  getRequestInstance: Function,
```

Comment:
We should probably make it clear that it requires a promise:

```js
getRequestInstance: () => Promise<*>
```

Reply:
done